cnx/src/lib.rs

```diff
@@ -141,8 +141,16 @@ pub use bar::Position;
 /// [`add_widget()`]: #method.add_widget
 /// [`run()`]: #method.run
 pub struct Cnx {
+    /// The position of the Cnx bar
     position: Position,
+    /// The list of widgets attached to the Cnx bar
     widgets: Vec<Box<dyn Widget>>,
+    /// The (x,y) offset of the bar
+    /// It can be used in order to run multiple bars in a multi-monitor setup
+    offset: (i16, i16),
```

Comment:
Instead of tuple can you make this as Struct instead ? Also, can you make it as optional too.

Reply:
Added the struct, though it seems more logical to use the fefault of 0,0, rather than making it an Option

Reply:
Yeah, that sounds better.